ui/src/api.ts

```diff
@@ -25,14 +25,25 @@ export async function getAllPapers() {
 }
 
 export async function papers(s2Ids: string[]) {
-  const data = await doGet(
-    axios.get("/api/v0/papers", {
-      params: {
-        id: s2Ids.join(",")
-      }
-    })
-  );
-  return (data || []) as Paper[];
+  const PAPER_REQUEST_BATCH_SIZE = 50;
```

Comment:
@jocelynzsun Can you add a comment here to describe why we need to batch the requests to the API (specifically, that it seems that in production, when more than about 50 papers are requested from the API, sometimes the API returns a 400?)